lib/puppet/provider/postgresql_conf/ruby.rb

```diff
@@ -75,7 +75,7 @@ def write_config(file, lines)
   # check, if resource exists in postgresql.conf file
   def exists?
     select = parse_config.select { |hash| hash[:key] == resource[:key] }
-    raise ParserError, "found multiple config items of #{resource[:key]} found, please fix this" if select.length > 1
+    raise Puppet::Error, "found multiple config items of #{resource[:key]} found, please fix this" if select.length > 1
```

Comment:
I think one `found` is enough for this sentence?

Reply:
indeed, updated it.